mongo/src/main/java/com/redhat/lightblue/mongo/crud/IterateAndUpdate.java

```diff
@@ -48,28 +59,39 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(IterateAndUpdate.class);
 
+    private final int batchSize = 64;
```

Comment:
I see this declared a second time. Maybe it should be defined in a single location?